ui/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -174,7 +174,7 @@ export default class PermissionConnect extends Component {
             className="permissions-connect__back"
             onClick={() => this.goBack()}
           >
-            <i className="fas fa-chevron-left" />
+            <i className="fas fa-chevron-left"></i>
```

Comment:
nit: Could use short hand for closed tag here
```suggestion
            <i className="fas fa-chevron-left" />
```